scripts/register_new_matrix_user

```diff
@@ -197,4 +202,10 @@ if __name__ == "__main__":
     else:
         secret = args.shared_secret
 
-    register_new_user(args.user, args.password, args.server_url, secret, args.admin)
+    if args.admin and args.regular_user:
+        print "Both --admin and --regular-user are set, choose either."
+        sys.exit(1)
```

Comment:
Rather than doing this manually, we should probably use the inbuilt argparse conflict functionality: https://docs.python.org/2/howto/argparse.html#conflicting-options